### DISCUSSION TITLE
Martins-Aquarium-will-not-run

### DISCUSSION THREAD

**Discussion by brandonjon26** [nss-day-cohort-47/C47-Discussions#21] [Help Me] [answered]
Name: Brandon Hill

Have you searched the other help tickets?
There are no tickets for my particular problem.

Describe your issue:
My code looks good to me, but nothing will run on my page. I'm also not getting any error messages.

List any error messages that appear: 
None :(

What have you googled?
Without any errors, I'm not exactly sure what I should google.

What have you tried? Tell us everything you have tried.
I cloned Brenda's repo to view our code side by side. I've also gotten with a few classmates to see if we could figure this out together.

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with:
https://github.com/brandonjon26/martins-aquarium

**Comment by heymonicakay** [comment]
@brandonjon26 Are you seeing your message , "main is loaded" , come up in your console? If not, investigate why...
Remember that JavaScript is case sensitive, so all of your file paths and variables must be consistent across your application. 
Once you have made changes, please push up to your repository so we can see what you've changed!

**Reply by brandonjon26** [reply]
Thank you!!! I'm gonna see if I can't work this out after Bryson's response

**Comment by BrysonMG** [comment, marked as answer]
Like Monica was hinting at, it looks like your main.js isn't running, and since it's the first domino in the chain, the rest aren't going through either. 

Look at your script tag in index.html, specifically at your source.

`<script type="module" scr="scripts/main.js"></script>`

you have "scr" instead of "src".

After you fix that, the main.js will load; however, you encounter more errors. Check out your various imports and exports and address any discrepancies you find. The error messages in dev tools should help you zero in on what's causing the snags.

**Reply by brandonjon26** [reply]
Thank you so much! I made the change and I'm definitely getting somewhere now. I appreciate you taking the time to check my code.
